server/base/src/main/java/org/apache/accumulo/server/rpc/ThriftServerType.java

```diff
@@ -43,7 +43,7 @@ private ThriftServerType(String name) {
   public static ThriftServerType get(String name) {
     // Our custom HsHa server is the default (if none is provided)
     if (StringUtils.isBlank(name)) {
-      return THREADED_SELECTOR;
+      return CUSTOM_HS_HA;
```

Comment:
Should this `return getDefault();` instead, so have the default specified in fewer places?